src/main/java/com/google/sps/data/Room.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.data;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.HashMap;
+import javax.servlet.http.HttpServletRequest;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.Gson;
+
+public class Room {
+    public static final Gson PARSER = new Gson();
+    private List<String> members;
+    private LinkedList<Message> messages;
+    private ArrayList<String> videoUrls;
```

Comment:
All the declarations should be of type List<...> ....

Why use LinkedList for messages, and Arraylist for video urls?